declarations/Threads.json

```
@@ -4,20 +4,28 @@
     "Terms of Service": {
       "fetch": "https://help.instagram.com/769983657850450?helpref=faq_content&cms_id=769983657850450",
```

We recommend removing unnecessary query string parameters, as they might impact the rendered page in unforeseen ways.

```suggestion
      "fetch": "https://help.instagram.com/769983657850450",
```

---

Thank you, Matti! Will fix it now